cmd/genji/shell/command.go

```diff
@@ -116,26 +134,36 @@ func runHelpCmd() error {
 	for _, c := range commands {
 		// spaces indentation for readability.
 		spaces := 25
-		indent := spaces - len(c.Name) - len(c.Options)
-		fmt.Printf("%s %s %*s %s\n", c.Name, c.Options, indent, "", c.Description)
+		indent := spaces - len(c.DisplayName) - len(c.Options)
+		fmt.Printf("%s %s %*s %s\n", c.DisplayName, c.Options, indent, "", c.Description)
 	}
 
 	return nil
 }
 
+func shouldDisplaySuggestion(name, in string) bool {
+	// input should be at least half the command size to get a suggestion.
+	d := levenshtein.ComputeDistance(name, in)
+	return d < (len(name) / 2)
+}
+
 // displaySuggestions shows suggestions.
 func displaySuggestions(in string) error {
 	var suggestions []string
 	for _, c := range commands {
-		d := levenshtein.ComputeDistance(c.Name, in)
-		// input should be at least half the command size to get a suggestion.
-		if d < (len(c.Name) / 2) {
+		if shouldDisplaySuggestion(c.Name, in) {
 			suggestions = append(suggestions, c.Name)
 		}
+
+		for _, alias := range c.Aliases {
+			if shouldDisplaySuggestion(alias, in) {
+				suggestions = append(suggestions, alias)
+			}
+		}
 	}
 
 	if len(suggestions) == 0 {
-		return fmt.Errorf("Unknown command %q. Enter \".help\" for help.", in)
+		return fmt.Errorf("unknown command %q. Enter \".help\" for help", in)
```

Comment:
This fixes a go-lint issue: `error strings should not be capitalized or end with punctuation or a newline`